05/Validator.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class Validator {
+
+    public static function string($value, $min = 1, $max = INF) {
+        $value = trim($value);
```

Comment:
Czy wstawienie tego tutaj to dobry pomysł? Przyjmujemy, że użytkownik podał " abc ", minimalna ilość znaków to 3, a max to 4. Wszystko działa! Potem zapisujesz dane w bazie danych i masz 5 znaków! Dlaczego?! Ten zapis nie modyfikuje danych na zewnątrz (to akurat dobrze) jednak moim zdaniem `trim()` powinien być na zewn. nie wew.